dbservice/checker.go

```diff
@@ -33,7 +34,15 @@ func CheckSizeLimit() {
 			size += len(v)
 		}
 		if size > DbSizeLimit {
-			return tx.DeleteBucket([]byte(dbBucketName))
+			err = tx.DeleteBucket([]byte(dbBucketName))
```

Comment:
do we have unit test for this logic?

Reply:
I changed some tests. We now have unit tests for the `CheckSizeLimit` and `DeleteBucket` functions. 
Can you take a look?